ui/lib/core/addon/components/overview-card.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="selectable-card is-rounded no-flex" data-test-selectable-card-container={{@cardTitle}}>
+  <div class="is-flex-between is-fullwidth card-details" data-test-selectable-card={{@cardTitle}}>
+    <h3 class="title is-5">{{@cardTitle}}</h3>
+    <LinkTo
+      class="has-icon-right is-ghost is-no-underline has-text-semibold"
+      @route={{@actionTo}}
+      @query={{hash itemType=@queryParam}}
```

Comment:
To make this truly generic, we should let the parent pass the entire query here, since other implementations may not have a use for the `itemType` query param
```suggestion
      @query={{@actionQuery}}
```